src/star/star_genomegenerate/test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -e
+
+## VIASH START
+## VIASH END
+
+#########################################################################################
+
+echo "> Prepare test data"
+
+cat > genome.fasta <<'EOF'
+>chr1
+TGGCATGAGCCAACGAACGCTGCCTCATAAGCCTCACACATCCGCGCCTATGTTGTGACTCTCTGTGAGCGTTCGTGGG
+GCTCGTCACCACTATGGTTGGCCGGTTAGTAGTGTGACTCCTGGTTTTCTGGAGCTTCTTTAAACCGTAGTCCAGTCAA
+TGCGAATGGCACTTCACGACGGACTGTCCTTAGCTCAGGGGA
+EOF
+
+cat > genes.gtf <<'EOF'
+chr1    example_source  gene    0    50   .   +   .   gene_id "gene1"; transcript_id "transcript1";
+chr1    example_source  exon    20   40   .   +   .   gene_id "gene1"; transcript_id "transcript1"; 
+EOF
+
+#########################################################################################
+
+mkdir star_index
```

Comment:
This is covered in the component script itself, right?

Reply:
Fixed the script.